osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/mappers/XMLGregorianCalendarToInstantConverter.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2020 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.shared.mappers;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.util.GregorianCalendar;
+
+import javax.xml.datatype.DatatypeConfigurationException;
+import javax.xml.datatype.DatatypeConstants;
+import javax.xml.datatype.DatatypeFactory;
+import javax.xml.datatype.XMLGregorianCalendar;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ma.glasnost.orika.MappingContext;
+import ma.glasnost.orika.converter.BidirectionalConverter;
+import ma.glasnost.orika.metadata.Type;
+
+public class XMLGregorianCalendarToInstantConverter extends BidirectionalConverter<XMLGregorianCalendar, Instant> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(XMLGregorianCalendarToInstantConverter.class);
+    private static final int SECONDS_PER_MINUTE = 60;
+
+    @Override
+    public XMLGregorianCalendar convertFrom(final Instant source, final Type<XMLGregorianCalendar> destinationType,
```

Comment:
Converting `Instant` values to `XMLGregorianCalendar` looks familiar from the `DeviceConverterHelper`, yet the implementation looks a little different.
Is the difference important, or would it be better to have more re-use of a single implementation?
Or should we wait until the story to deal with date and time helpers gets picked from the backlog?

Reply:
Removed the convert methods from the `DeviceConverterHelper`. 
Orika seems to contain a built-in `DateToXmlGregorianCalendarConverter` class and mapping between `Instant` and `XMLGregorianCalendar` couldl be done by this converter class.